Environment_Engine/Modify/SetPanelType.cs

```diff
@@ -119,5 +119,111 @@ public static List<Panel> SetPanelType(this List<Panel> panels, PanelType type)
 
             return panels;
         }
+
+        [Description("Returns the floor panels of a space represented by Environment Panels")]
+        [Input("panelsAsSpace", "A collection of Environment Panels that represent a closed space")]
+        [Output("floorPanels", "BHoM Environment panel representing the floor of the space")]
+        public static List<Panel> FloorPanels(this List<Panel> panelsAsSpace)
```

Comment:
Now the method is moved, can we call it `SetFloorPanels` instead - just a last thought I've had

Reply:
done